Assets/MixedRealityToolkit/_Core/Inspectors/Utilities/Lines/DataProviders/EllipseLineDataProviderInspector.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Inspectors.Utilities.Lines.DataProviders
+{
+    [CustomEditor(typeof(EllipseLineDataProvider))]
+    public class EllipseLineDataProviderInspector : BaseMixedRealityLineDataProviderInspector
+    {
+        private SerializedProperty resolution;
+        private SerializedProperty radius;
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            // Bump up the resolution, in case it's too low
+            if (LinePreviewResolution < 32)
```

Comment:
replace 32 with const (ex: MinLineResolution)

Reply:
Why? we only use this value in one place.

Reply:
mostly because I am not a fan of magic number :)

Reply:
Would 42 be better?

Reply:
nope. 43. hahaha!